apps/backend/src/schema/schema.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from 'zod';
+
+const addressSchema = z.object({
+    address: z.string(),
+    city: z.string(),
+    state: z.string(),
+    zipcode: z.number(),
+})
+
+// Part of form to be filled out by the child's parent/legal guardian
+const guardianFormSchema = z.object({
+    childsName: z.string(),
+    ssn: z.number(),
+    dob: z.string(),
+    gender: z.string(),
+    guardianName: z.string(),
+    address: addressSchema,
+    phone: z.number(),
```

Comment:
I think it makes more sense to store phone numbers as strings with some regex checks